Assets/helloconsole.cs

```diff
@@ -7,7 +7,8 @@ public class helloconsole : MonoBehaviour
     // Start is called before the first frame update
     void Start()
     {
-        Debug.Log("Hello World - Jack B");
+        // write to console
+        Debug.Log("919946");
```

Comment:
Not happy to accept change to the print statement, please change back to original. The comment is fine.

Reply:
Ok